client/utils/calculatePosition.js

```diff
@@ -1,18 +1,30 @@
-export default function ({ newPosition, items, isFirstChild, insert = false }) {
+export default function ({
```

Comment:
Since we have more actions now we should consider changing signature and instead insert and prepend booleans have action enum with (ADD_BEFORE, ADD_AFTER, and REORDER values; export enum as well). Thoughts? (It seams like the code would be more readable as well).